src/Common/Process.cpp

```diff
@@ -17,80 +17,89 @@
 
 #include "Common/Process.h"
 
+#if defined( _WIN32 )
 #include "Common/Platform/Win32/Win32Error.h"
 #include "Common/Platform/Win32/Win32ProcessInfo.h"
```

Comment:
No need to if-def local headers, I prefer putting the if-def inside the headers themselves to all files can safely be included on all platforms

Reply:
What about the UNIX system headers? Leave them in an ifdef?

Reply:
Yep!